css-flexbox/flex-wrap.html

```diff
@@ -9,11 +9,15 @@
   <link rel="stylesheet" href="pokemon.css">
   <style>
     /* your styles here 👇 */
-    #card-deck {
+    .card-wrapper {
       display: flex;
       flex-wrap: wrap;
       flex-basis: calc(100% / 6);
```

Comment:
Please see above comment regarding `flex-basis` and applying `display: flex` to `#card-deck`